Gemfile.lock

```diff
@@ -72,15 +72,14 @@ GEM
       railties (>= 5.0.0)
     faker (2.18.0)
       i18n (>= 1.6, < 2)
-    faraday (1.6.0)
+    faraday (1.5.1)
```

Comment:
Unusual that your gem versions are different...

Reply:
yeah, there is a bunch of weird gem stuff going on right now